### ISSUE TITLE
No Language Attribute Error for Youtube

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#12]
![image](https://cloud.githubusercontent.com/assets/15111528/10756612/3b13c728-7cf9-11e5-92a5-49d5e555a928.png)

When I check with the Vision Australia tool bar, for the Language Attribute, error displayed (attached screenshot). @christosptr Can you please advise if this needsa language of part?

**Comment by christosptr** [comment]
@HerinHentry 

I have specified the language in the the video I have uploaded. I am not aware of a specification that need to have the language of part. What I know about the language of part is that for example when you are in an English website and you refer to a French name then you will need the language of part so that screen readers can read the French name correctly.

I have not come across a specification that requires a language of part for a video
